lib/chatops_deployer/app.rb

```diff
@@ -26,10 +26,12 @@ class App < Sinatra::Base
       json = JSON.parse(request.body.read)
       post_url = json['callback_url']
       clean = json['clean'] != 'false'
+      host = json['host'] ||= 'github.com'
```

Comment:
I don't think `||=` is required here. just `||` would do. Otherwise this would set the `json['host']` param to 'github.com' if it's `nil`. Is that what you want?